src/server/endpoints/friendHandler.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const HTTP_STATUSES = require('../modules/httpStatuses');
+const friendsCollection = require('../collections/friendsDatabase');
+const usersCollection = require('../collections/usersDatabase');
+const {getAccessToken} = require('../modules/tokenHandler');
+
+function findFriends(idArray, res) {
+  usersCollection.findUsers(idArray, (err, result) => {
+    if (err !== null) {
+      res.status(HTTP_STATUSES.SERVER_ERROR).json({errorType: 'serverError'});
+      return;
+    }
+    let friendsInfo = [];
+
+    result.forEach(function(user) {
+      let newUser = {};
```

Comment:
stop this object creation thing. You dont really need to do this. you can write:
```
return {
 ....
}
```